compiler/plugins/target/AMD-AIE/aie/AIENormalizeAddressSpaces.inc

```diff
@@ -0,0 +1,106 @@
+/*===- TableGen'erated file -------------------------------------*- C++ -*-===*\
```

Comment:
Why not copy the tablegen file instead?

Reply:
We don't use `def Pat` anywhere right? Seemed weird to introduce it here just a for a thing that adds a default address space. Really I should just rewrite it (I'll do that now).